resources/en-US/training/t-w.training.bxb

```diff
@@ -0,0 +1,5 @@
+train (t-wob48d5x7x7vpbu85hzh6lazg) {
+  utterance ("[g:CommitOrder:continue,r:UpdateOrder] Remove the large {[g:RemovedItem] (v-neck)[v:SearchTerm]} and the small {[g:RemovedItem2] (v-neck)[v:SearchTerm]}")
```

Comment:
these sizes don't need tags?

Reply:
no - can't search current items with size (it'll confuse planner)